app/src/components/market/sections/market_create/steps/items/create_scalar_market.tsx

```diff
@@ -125,6 +120,52 @@ export const CreateScalarMarket = (props: Props) => {
     upperBound,
   } = props
 
+  let lowerBoundNumber: string
+  let startingPointNumber: string
+  let upperBoundNumber: string
+
+  if (startingPoint !== null) {
+    startingPointNumber = formatBigNumber(startingPoint, 18, 2)
+  }
+
+  if (lowerBound !== null) {
+    lowerBoundNumber = formatBigNumber(lowerBound, 18, 2)
+  }
+
+  if (upperBound !== null) {
+    upperBoundNumber = formatBigNumber(upperBound, 18, 2)
+  }
+
+  const [lowerBoundError, setLowerBoundError] = useState('')
+  const [startingPointError, setStartingPointError] = useState('')
+  const [upperBoundError, setUpperBoundError] = useState('')
+
+  const handleLowerBoundError = (value: BigNumberInputReturn) => {
+    Number(value.formattedValue) <= 0
+      ? setLowerBoundError('Value must be greater than 0')
+      : Number(value.formattedValue) > Number(upperBoundNumber) && Number(upperBoundNumber) !== 0
+      ? setLowerBoundError(`Value must be less than ${upperBoundNumber}`)
+      : setLowerBoundError('')
```

Comment:
Now if we're just doing `BigNumber` comparisons here, we can just do e.g. `value.lt(Zero) ? setLowerBoundError`.

For reference: https://docs.ethers.io/v5/api/utils/bignumber/

Reply:
`value` here is of type `BigNumberInputReturn` which `lt()` method doesn't exist on. I think this is the ideal way of comparing values?

Reply:
those docs suggest you can typecast  the value using `BigNumber.from()` but it only gave me errors? i tried the syntax they use as well `ethers.BigNumber.from(n)` is it possible?

Reply:
> `value` here is of type `BigNumberInputReturn` which `lt()` method doesn't exist on. I think this is the ideal way of comparing values?

Yeah, should actually be `value.value` here to get the actual `BigNumber` value

> those docs suggest you can typecast the value using `BigNumber.from()` but it only gave me errors? i tried the syntax they use as well `ethers.BigNumber.from(n)` is it possible?

My bad, I shared the v5 docs when we're actually on v4: https://docs.ethers.io/v4/api-utils.html#big-numbers